### ISSUE TITLE
The People filter on the Mobile App doesn't show all photos of the person.

### ISSUE THREAD

**Issue by RaulGw4** [immich-app/immich#11095]
### The bug

When you click on a person in Immich, it shows far fewer photos compared to using the Web App.

### The OS that Immich Server is running on

Truenas Scale

### Version of Immich Server

v1.108.0

### Version of Immich Mobile App

1.108.0 build.148

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
N/A
```

### Your .env content

```Shell
N/A
```

### Reproduction steps

```bash
1. Go to the "Search" tab.
2. Click on a person you know has over 1500 photos.
3. Scroll all the way down in the Mobile App and compare them to the photos in the Web App. (In my case, in the Web App for a person, I have photos going back to 2007, but when I scroll down in the Mobile App, the oldest photo is from 2012, which is incorrect.)
```

### Relevant log output

```shell
N/A
```

### Additional information

I am using a Nothing Phone 2 running Android 14.
There are no errors in the logs.

**Comment by xuecqcn** [comment]
me too

**Comment by mohammadhosseinhosseini** [comment]
It is the same for me. I'm using iOS 17. I can only access the full list from the web view.

**Comment by SimKaiLong** [comment]
I'm getting this as well on the Android app. Running v1.110.0 for server and app version 1.110.0 build.151

**Comment by andyfranco** [comment]
I have the same issue on v1.111 on iOS 17. Seems like there must be a max limit it can show. I have to use the browser on my phone to view older photos for a tagged person.

**Comment by Weltherrscher** [comment]
Same here, Sony Xperia 10 V, on one person are about 7k pictures back to 2016, but i can only see until july 2023...

**Comment by alextran1502** [comment]
Solved via #11597
